index.ts

```diff
@@ -402,7 +447,7 @@ export class OAuth2AuthCodePKCE {
         default:
           break;
       }
-      return Promise.reject(error);
+      return Promise.reject(toErrorClass(error));
```

Comment:
This may be a breaking change depending on usage, but I thought it should be consistent with other errors.

Reply:
No this is probably the correct thing to do - thanks!